.env.example

```diff
@@ -1,2 +1,3 @@
-REACT_APP_AAD_CLIENT_ID=276fd164-c30c-485f-bd76-a8f3b5bcac74
+REACT_APP_AAD_CLIENT_ID=5dd59e6d-1cb1-4ec7-8e26-79a6dd9859b4
```

Comment:
This is the client id of the app registration: `Beacons Registry Back Office Local Development` which allows users to login using any Azure AD account